src/homework14/Box.java

```diff
@@ -0,0 +1,64 @@
+package homework14;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class Box <T>{
+    public List<Fruit> boxList = new ArrayList<>();
+    public String insideType;
+    public float totalWeight = 0;
+
+    public Box() {
+        System.out.println("New box was created!");
+    }
+    //2.4. Реалізувати метод додавання фрукта (1 од.) до коробки.
+    public void add(Fruit fruit) {
```

Comment:
`public void add(Fruit fruit)` means that the user can put `Apple` and `Orange` into the box - this is incorrect.
The correct way is `public void add(T fruit)`.